CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Drop support for Ruby < 2.4 and Rails < 5.2.
```

Comment:
Could you remove the changelog changes? I'd like to queue up everything and do a MV bump since we are dropping old ruby / rails versions.

Reply:
Actually I can take care of this.